packages/manager/apps/pci-rancher/src/data/hooks/useFormattedPrices/useFormattedPrices.tsx

```diff
@@ -10,8 +11,9 @@ export const useFormattedRancherPrices = (
   plans: RancherPlan[],
   pricing?: TRancherPricing[],
 ) => {
-  const { getFormattedHourlyCatalogPrice } = useCatalogPrice(5);
-  const { getFormattedMonthlyCatalogPrice } = useCatalogPrice(2);
+  const { getFormattedCatalogPrice: hourly } = useCatalogPrice(5);
+  const { getFormattedCatalogPrice: monthly } = useCatalogPrice(2);
```

Comment:
Maybe can add constant for better readability ?

```suggestion
const ID_CATALOR_FOR_HOUR_PRICE = 5;
const ID_CATALOR_FOR_MONTHLY_PRICE = 2;

  const { getFormattedCatalogPrice: hourlyPrice } = useCatalogPrice(ID_CATALOR_FOR_HOUR_PRICE);
  const { getFormattedCatalogPrice: monthlyPrice } = useCatalogPrice(ID_CATALOR_FOR_MONTHLY_PRICE);
```